src/main/_docs/devops/devops-runtime-stacks.md

```diff
@@ -41,7 +41,10 @@ Che provides a form that can be used to write a recipe directly or copied/pasted
 
 # Stack Administration  
 ## Stack Loading
-In 5.x, we introduced an underlying database for storing product configuration state, including the state of stacks and templates. In previous versions, we primarily allowed stack configuration through a `stacks.json` object file that was in the base of a Che assembly. The `stacks.json` object file is still there, and if you provide any stack definitions within it, they will be loaded (and override!) any stacks in the database whenever Che starts. We will be removing support for the JSON configuration approach in upcoming releases as it is error prone.
+
+Stacks are loaded from a JSON file that is packaged into resources of a special component deployed with a workspace makes. This JSON isn't exposed to users and stack management is performed in User Dashboard (that uses REST API).
+
+Stacks are loaded from a JSON file only when the database is initialized, i.e. when a user first stats Che. This is the default policy that can be changed. To keep getting updates with new Che stacks, set `CHE_PREDEFINED_STACKS_RELOAD__ON__START=true` in `che.env`. When set to true, stack.json will be used to update Che database, each time Che server starts.
 
```

Comment:
Even if it is not clean, it will still be needed for those who are creating custom Che. So either  eclipse/che-archetypes#20 was a pre-requisite for merging https://github.com/eclipse/che/pull/5234 or we provide documentation to explain how to handle that until eclipse/che-archetypes#20 is fixed.

Also "update" will mislead the user. The "update" must be completely explicit and clear to the user - so that he will know that when configuring the property, the system will wipeout all stacks currently in the product. Including those that he added manually. So he will cautiously configure that property.

It must also explain to the user how he can get stacks from a new version of Che and not loose the one he added. Here again, even if the way we handle that is not nice, etiher we have fixes that are coming and we make them pre-requisite for merging https://github.com/eclipse/che/pull/5234 or we provide documentation to explain how it can be handled.